src/zeroconf/_services/info.py

```diff
@@ -503,23 +502,23 @@ def dns_addresses(
                 class_,
                 ttl,
                 ip_addr.packed,
-                created=created,
+                created=0,
             )
             for ip_addr in self._ip_addresses_by_version_value(version_value)
         ]
 
-    def dns_pointer(self, override_ttl: Optional[int] = None, created: Optional[float] = None) -> DNSPointer:
+    def dns_pointer(self, override_ttl: Optional[int] = None) -> DNSPointer:
         """Return DNSPointer from ServiceInfo."""
         return DNSPointer(
             self.type,
             _TYPE_PTR,
             _CLASS_IN,
             override_ttl if override_ttl is not None else self.other_ttl,
             self._name,
-            created,
+            0,
         )
```

Comment:
We could add a `cached_dns_pointer`, `cached_dns_service`, `cached_dns_text`, `cached_`... property

Ideally we only generate these records once the service info is added to the registry so the query handler doesn't create these records over and over and over again